app/Console/Commands/PostArticleToTwitter.php

```diff
@@ -15,15 +15,10 @@ final class PostArticleToTwitter extends Command
 
     private $notifiable;
 
-    public function __construct(AnonymousNotifiable $notifiable)
+    public function handle(AnonymousNotifiable $notifiable): void
```

Comment:
Shifting constructor injection to handle method injection as this is a check in Enlightn. All command constructors are called when the CLI boots so `handle` method injection allows lazy loading.